### PR TITLE
integration test for list -f json

### PR DESCRIPTION
**Pull request by YaoZengzeng**
Signed-off-by: Yao Zengzeng <yaozengzeng@zju.edu.cn>

### CODE REVIEW

File: integration-test/list_test.go [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+	"encoding/json"
+	"strings"
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/go-check/check"
 )
 
+type cList struct {
+	// ID is the container ID
```
Comment (gao-feng) [inline integration-test/list_test.go]: Is this structure from containerState of runc?
I suggest we use State structure from oci. Create new structure combinded with the different fields and State.
Such as
```
type containerState struct {
    specs.State
    Bundle string `json:"bundle"` 
}
```

File: integration-test/list_test.go [hunk 2]
```diff
@@ -36,3 +51,49 @@ func (s *RunVSuite) TestListSleep(c *check.C) {
 	c.Assert(out, checker.Contains, ctrName)
 	<-exitChan
 }
+
+func (s *RunVSuite) TestListSleepJson(c *check.C) {
+	//TODO: enable this after fixing!!!
+	//c.Skip("enable this after fixing")
```
Comment (gao-feng) [inline integration-test/list_test.go]: what's this meaning?

Comment (YaoZengzeng) [inline integration-test/list_test.go]: Hmmm... I don't know as well.
I just find every test case has it at the beginning, so I write it there.

**Comment by YaoZengzeng** [comment]
Ready for review

**Comment by gao-feng** [comment]
lgtm

**Pull request merged**